src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -0,0 +1,445 @@
+        logger.warning("If sync token '%s' already exists, it properties will be overwritten", sync_token_name)
+        Token = cmd.get_models('Token')
+        poller = token_client.create(
+            resource_group_name,
+            registry_name,
+            sync_token_name,
+            Token(
+                scope_map_id=sync_scope_map.id,
+                status="enabled"
+            )
+        )
+
+        token = LongRunningOperation(cmd.cli_ctx)(poller)
+        return token.id
+    except ValidationError as e:
+        raise CLIError(e)
+
+
+def _get_descendancy(family_tree, parent_id):
+    childs = family_tree[parent_id]['childs']
+    result = []
+    for child_id in childs:
+        result = [family_tree[child_id]["registry"]]
+        descendancy = _get_descendancy(family_tree, child_id)
+        if descendancy:
+            result.extend(descendancy)
+    return result
+
+
+# region connected-registry install subgroup
+def acr_connected_registry_install_info(cmd,
+                                        client,
+                                        connected_registry_name,
+                                        registry_name,
+                                        resource_group_name=None):
+    return _get_install_info(cmd, client, connected_registry_name, registry_name, False, resource_group_name)
+
+
+def acr_connected_registry_install_renew_credentials(cmd,
+                                                     client,
+                                                     connected_registry_name,
+                                                     registry_name,
+                                                     resource_group_name=None):
+    return _get_install_info(cmd, client, connected_registry_name, registry_name, True, resource_group_name)
+
+
+def _get_install_info(cmd,
+                      client,
+                      connected_registry_name,
+                      registry_name,
+                      regenerate_credentials,
+                      resource_group_name=None):
+    registry, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name)
+    connected_registry = acr_connected_registry_show(
+        cmd, client, connected_registry_name, registry_name, resource_group_name)
+    parent_gateway_endpoint = connected_registry.parent.sync_properties.gateway_endpoint
+    parent_id = connected_registry.parent.id
+    sync_token_name = connected_registry.parent.sync_properties.token_id.split('/tokens/')[1]
+    if parent_id:
+        parent = parent_id.split('/connectedRegistries/')[1]
+        parent = acr_connected_registry_show(
+            cmd, client, parent, registry_name, resource_group_name)
+        parent_registry_endpoint = parent.login_server.host
+    else:
+        parent_registry_endpoint = registry.login_server
+
+    connected_registry_login_server = connected_registry.login_server.host
```

Comment:
I remember we agree to always provide a place holder string with sample and aks.ms link for connected_registry_login_server install parameter

Reply:
my bad I thought it was only when the value was null.